definitions/pipelines/rnaseq.cwl

```diff
@@ -129,10 +132,16 @@ steps:
             bam: merge/merged_bam
         out:
             [indexed_bam]
+    mark_dup:
+        run: ../tools/mark_duplicates_and_sort.cwl
+        in:
+            bam: merge/merged_bam
+        out:
+            [sorted_bam, metrics_file]
     stringtie:
         run: ../tools/stringtie.cwl
         in:
-            bam: merge/merged_bam
+            bam: mark_dup/sorted_bam
```

Comment:
Did you verify that stringtie ignores duplicate marking?  (either in their docs/online or by testing it yourself?)

Reply:
I couldn't find any literature of stringtie that says it ignores duplicate marking, but i ran a test and the outputs with/without markdup were the same.